edx/edx-enterprise
# [ENT-986] Update DSC text to match new enterprise welcome text.

**Description:** This changes the DSC text to match what is now on several other pages.

**JIRA:** https://openedx.atlassian.net/browse/ENT-986

**Merge deadline:** Anytime

**Testing instructions:**

1. Go to the DSC page for a customer that has enabled DSC, and notice this change.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**
